examples/box_game/box_game.rs

```diff
@@ -123,7 +123,7 @@ pub fn setup_system(
                 // ...add visual information...
                 PbrBundle {
                     mesh: mesh.clone(),
-                    material: materials.add(color.into()),
+                    material: materials.add(StandardMaterial::from(color)),
```

Comment:
Is this just a style thing?

Personally, I find it a bit verbose.

Reply:
Yeah I had an issue where it was complaining it didn't know what particular `Into` implementation to use. I threw this in just to get around it without too much investigation.

Reply:
I don't have a strong opinion, but I am leaning towards liking it, since it is explicit about the magic bevy type that goes into this parameter.